ui/desktop/app/styles/_notify.scss

```diff
@@ -88,8 +88,9 @@
   }
 }
 
-@media only screen and (max-width: 680px) {
+@media only screen and (width <= 680px) {
   .ember-notify-cn {
+    background-color: #fff;
```

Comment:
None blocking comments:
- What is the purpose of adding white as background color?
- Shouldn't we use a HDS color token rather than hex color value? perhaps `foreground-high-contrats` [color palette link](https://helios.hashicorp.design/foundations/colors?tab=palette).

Reply:
@calcaide totally forgot to remove that, just doing some testing